modules/client-handler/src/main/java/org/apache/ignite/client/handler/requests/table/ClientTableGetRequestV2.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.handler.requests.table;
+
+import java.util.concurrent.CompletableFuture;
+import org.apache.ignite.internal.client.proto.ClientMessagePacker;
+import org.apache.ignite.internal.client.proto.ClientMessageUnpacker;
+import org.apache.ignite.internal.table.TableViewInternal;
+import org.apache.ignite.table.IgniteTables;
+import org.apache.ignite.table.QualifiedName;
+
+/**
+ * Client table retrieval request.
+ */
+public class ClientTableGetRequestV2 {
+    /**
+     * Processes the request.
+     *
+     * @param in     Unpacker.
+     * @param out    Packer.
+     * @param tables Ignite tables.
+     * @return Future.
+     */
+    public static CompletableFuture<Void> process(
+            ClientMessageUnpacker in,
+            ClientMessagePacker out,
+            IgniteTables tables
+    ) {
+        QualifiedName qualifiedName = in.unpackQualifiedName();
```

Comment:
Let's avoid string parsing here, and se `QualifiedNameHelper.fromNormalize`.
We've already done name validation+normalization on client side.

btw: any error in this call may lead to hanging. The call should be wrapped into future or there should be a try-catch block.